pkg/handler/handler_test.go

```diff
@@ -211,18 +211,27 @@ func TestCacheError(t *testing.T) {
 }
 
 func TestCachePreviousValue(t *testing.T) {
-	prevVal := []byte("previous value")
+	origVal := []byte("previous value")
 	cacheObj := cache{
```

Comment:
Maybe you should call the `New` function:
```golang
cache{}.New(func() ([]byte, error) {
...
```

Reply:
Gotcha, done